### PR TITLE
Tests for editRecord

### PR DESCRIPTION
**Pull request by albarivas**
### What does this PR do?
Adds tests for editRecord

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### Functionality Before

editRecord didn't have tests

### Functionality After

editRecord didn't has tests

### CODE REVIEW

File: force-app/test/jest-mocks/schema.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const Account = {
```
Comment (muenzpraeger) [inline force-app/test/jest-mocks/schema.js:1]: Love it!!

File: force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            );
+        });
+    });
+
+    it('displays the correct error toast message when there is an error updating a record', () => {
+        // Create initial element
+        const element = createElement('c-edit-record', {
+            is: EditRecord
+        });
+        element.recordId = '0019A00000E8zAWQAZ';
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js:137]: Let's make this a class level const.

File: force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        expect(recordForm.objectApiName).toStrictEqual(ACCOUNT_OBJECT);
+        expect(recordForm.fields).toStrictEqual([
+            ACCOUNT_NAME_FIELD,
+            ACCOUNT_TYPE_FIELD,
+            ACCOUNT_PHONE_FIELD,
+            ACCOUNT_EMPLOYEES_FIELD
+        ]);
+    });
+
+    it('displays the correct success toast message when a record is created successfully', () => {
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js:38]: We should group the toasts into "success" and "error" testsuites. 

`c-edit-record` => `shows success toastmessage` => `when a record is created`

Comment (albarivas) [reply force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js]: done

File: force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js (hunk 2 above)
Comment (albarivas) [reply force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js]: done

File: force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        expect(recordForm.objectApiName).toStrictEqual(ACCOUNT_OBJECT);
+        expect(recordForm.fields).toStrictEqual([
+            ACCOUNT_NAME_FIELD,
+            ACCOUNT_TYPE_FIELD,
+            ACCOUNT_PHONE_FIELD,
+            ACCOUNT_EMPLOYEES_FIELD
+        ]);
+    });
+
+    describe('show success toast message', () => {
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js:40]: `shows`

File: force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+                expect(handler).toHaveBeenCalled();
+                expect(handler.mock.calls[0][0].detail.variant).toBe('success');
+                expect(handler.mock.calls[0][0].detail.message).toBe(
+                    'Account updated'
+                );
+            });
+        });
+    });
+
+    describe('show error toast message', () => {
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js:105]: `shows`

File: force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+const RECORD_ID = '0019A00000E8zAWQAZ';
+
+describe('c-edit-record', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+    it('shows a lightning-record-form initialized', () => {
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/editRecord/__tests__/editRecord.test.js:19]: Newline before

**Pull request merged**